packages/storybook-addon-designs/tab.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path')
+
+const HtmlPlugin = require('html-webpack-plugin')
+const config = require('./preset')
+
+module.exports = {
+  ...config,
+  managerEntries(entries = []) {
```

Comment:
You can get addon options via 2nd argument like this:

```js
// our preset.js
module.exports = {
  managerEntries(entries = [], options = {}) {
    // ...
  }
}

// users' .storybook/main.js
module.exports = {
  addons: ['storybook-addon-designs', { foo: 'bar' }]
}
```

So I think adding an addon option that decides where to render the addon (Panel or Tab) is simple and easy to use instead of creating separate a preset file 👍

Reply:
```
addons: [
  {
    name: 'storybook-addon-designs',
    options: { foo: 'bar' }
  }
]
```

Reply:
I could not find this documented anywhere! I'll get this implemented tomorrow!